content/docs/user_guide/kubernetes/wordpress-quick-start.md

```diff
@@ -0,0 +1,247 @@
++++
+title = "GKE Sandbox Quickstart"
```

Comment:
I'm a bit wary of putting GKE docs on the gVisor website. Do you think this provides value over the ["Enabling GKE Sandbox"](https://cloud.google.com/kubernetes-engine/docs/how-to/sandbox-pods#enabling) and ["Running an application in a sandbox"](
https://cloud.google.com/kubernetes-engine/docs/how-to/sandbox-pods#sandboxed-application) sections in the GKE docs?

Maybe updating the GKE Sandbox docs to have a Service and allow verification by opening an outside connection would be better?

Reply:
Let's make a "tutorials" section and put this there. I'm worried about the expectation of maintaining this doc in the same format. Howtos or tutorials don't necessarily have that expectation and it's clearer what it's for.

Reply:
Done.